README.md

```diff
@@ -2,7 +2,7 @@
 
 SEO for [ApostropheCMS](http://apostrophecms.org/).
 
-Add useful meta fields to `apostrophe-custom-pages`, `apostrophe-pieces`, and `apostrophe-global`.
+Add useful meta fields to all pages and pieces.
```

Comment:
... Which address SEO needs, such as being able to distinguish the `title` tag from the title shown in navigation links, and being able to set the meta description tag.